asf/sam/include/sam4e/sam4e16c.h

```diff
@@ -250,7 +250,7 @@ void WDT_Handler        ( void );
  */
 
 #define __CM4_REV              0x0000 /**< SAM4E16C core revision number ([15:8] revision number, [7:0] patch number) */
-#define __MPU_PRESENT          0      /**< SAM4E16C does not provide a MPU */
+#define __MPU_PRESENT          1      /**< SAM4E16C does not provide a MPU */
```

Comment:
if you are going to change this, how about fixing the comment as well.

Reply:
Ahaha I can't believe, fixed!